lib/WebSocketConnection.js

```diff
@@ -106,7 +106,12 @@ function WebSocketConnection(socket, extensions, protocol, maskOutgoingPackets,
     this.socket.setNoDelay(this.config.disableNagleAlgorithm);
 
     // Make sure there is no socket inactivity timeout
-    this.socket.setTimeout(0);
+    this.socket.setTimeout(this.config.keepaliveInterval * 2.1);
```

Comment:
There is a check for keepaliveInterval on line 116.

Reply:
Added a check here as it is part of a different setup.